app/static/pages/partnerships/Partnerships.js

```diff
@@ -32,14 +32,16 @@ oppiaFoundationWebsite.controller(
           email: email,
           comment: comment,
         }).then(function() {
+          ga('send', 'event', 'Submit Form', 'Submit Partnership form', 'Partnerships');
+
           $mdDialog.show(
-            $mdDialog.alert()
-              .clickOutsideToClose(true)
-              .title('Thank you!')
-              .textContent(THANKYOU_MESSAGE)
-              .ariaLabel('Thank you dialog')
-              .ok('Got it!')
-              .targetEvent(evt)
+          $mdDialog.alert()
```

Comment:
To be clear, only lines 35 and 37 needed to be deindented. This and the following ones should be indented because they are in .show(). (Take a look at the previous version of the code -- it was correct there.)